деплой3

Сделай так, чтобы диплой работал.

[Deploy to VDS](https://github.com/mastint777/1/actions/workflows/deploy-to-vds.yml)
Merge pull request #11 from mastint777/issue-10-02dd62431f7a #5
Jobs
Run details
Annotations
1 error
[deploy](https://github.com/mastint777/1/actions/runs/19127304066/job/54659963165)
failed 18 minutes ago in 5s
Search logs
1s
1s
0s
Run appleboy/ssh-action@v1.2.0
Run echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
Run entrypoint.sh
Will download drone-ssh-[1](https://github.com/mastint777/1/actions/runs/19127304066/job/54659963165#step:3:1).8.0-linux-amd64 from https://github.com/appleboy/drone-ssh/releases/download/v1.8.0
======= CLI Version =======
Drone SSH version 1.8.0
===========================
2025/11/06 06:48:58 dial tcp: lookup tcp/6f02: unknown port
Error: Process completed with exit code 1.
0s

Сейчас выдает такую ошибку.

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -1,5 +1,0 @@
-Issue to solve: https://github.com/mastint777/1/issues/12
-Your prepared branch: issue-12-c89c58d7d544
-Your prepared working directory: /tmp/gh-issue-solver-1762412999012
-
-Proceed.

